bot/bot_interaction.py

```diff
@@ -0,0 +1,27 @@
+from typing import Final
+
+import telebot
+from telebot import types
```

Comment:
since you already imported _telebot_ package, remove this unnecessary import (later explicitly use _types_ module from _telebot_ package)